addons/mail/static/tests/qunit_suite_tests/components/composer_tests.js

```diff
@@ -1359,11 +1192,7 @@ QUnit.test('current partner notify is typing to other thread members', async fun
         },
     });
 
-    document.querySelector(`.o_ComposerTextInput_textarea`).focus();
-    document.execCommand('insertText', false, "a");
-    document.querySelector(`.o_ComposerTextInput_textarea`)
-        .dispatchEvent(new window.KeyboardEvent('keydown', { key: 'a' }));
-
+    insertText('.o_ComposerTextInput_textarea', 'a');
```

Comment:
missing await (already before) maybe not visible but this is bad (uncaught promise that is hidden because test ends before / memory leak / etc)